source/Pict-Content-Assignment.js

```diff
@@ -241,9 +248,9 @@ class PictContentAssignment extends libFableServiceBase
 
 	addClass(pAddress, pClass)
 	{
-		if (this.customAddClassFunction)
+		if (this.customSetClassFunction)
 		{
-			return this.customAddClassFunction(pAddress, pClass);
+			return this.customSetClassFunction(pAddress, pClass);
```

Comment:
hm I don't think we return a value for the other assignment functions, so we shouldn't do it here either.

In addition, this custom case is now inconsistent from the JQuery and Document cases below it.

Reply:
@kamaurra-headlight can you clarify this statement? I just followed the same pattern for Read Attribute